Makefile

```diff
@@ -46,16 +49,47 @@ develop: publish-build
 	docker-compose logs -f
 
 # Clean
+clean: TAG := latest
 clean:
 	docker-compose stop
 	docker-compose down --remove-orphans --volumes --rmi all
 	docker-compose rm -svf
-	docker rmi --force openwisp/openwisp-base:latest \
-				openwisp/openwisp-base:intermedia-system \
-				openwisp/openwisp-base:intermedia-python \
+	docker rmi --force $(DOCKER_USERNAME)/openwisp-base:$(TAG) \
+				$(DOCKER_USERNAME)/openwisp-base:intermedia-system \
+				$(DOCKER_USERNAME)/openwisp-base:intermedia-python \
+				`docker images -f "dangling=true" -q` || true
+
+release-clean: clean
+	docker-compose rm -svf
+	docker rmi --force $(DOCKER_USERNAME)/openwisp-base:$(TAG) \
+				$(DOCKER_USERNAME)/openwisp-base:intermedia-system \
+				$(DOCKER_USERNAME)/openwisp-base:intermedia-python \
 				`docker images -f "dangling=true" -q` || true
 
 # Publish
+publish: TAG := latest
 publish: publish-build develop-runtests
```

Comment:
I know this wasn't added as part of this pull but isn't the redundant?  Seems like it should only call `develop-runtests`

Reply:
Yeah I'm not forgetting this part too, thanks for reminding :smile: